packages/ui-signer/src/Modal.tsx

```diff
@@ -5,7 +5,10 @@
 import { ApiProps } from '@polkadot/ui-react-rx/types';
 import { I18nProps, BareProps } from '@polkadot/ui-app/types';
 import { QueueTx, QueueTx$Id, QueueTx$MessageSetStatus, QueueTx$Result, QueueTx$Status } from './types';
+// TODO - reuse instead of obtaining from app-transfer
```

Comment:
As per other comments, the dep needs to be flipped - app-* can depend on ui-*, however ui-* should never depend on app-*.